cb-talent-development/employer-style-base
# EM-1308: Header Large Basic to Header Small (ESB)

**Purpose:**

> As a designer or a front end developer, I'd like the pages that use the "header_large_basic" header partial to henceforth use the "header_small" partial (which might need to be created) so that going forward we have:
> - a defined set of headers
> - the same language for this header type
> - generic, reusable components when we want to use this header type again

This PR introduces the `header--small()` mixin and refactors the `header-hero()` mixin to use what was pulled from `header--large-basic` property found on the employer side.

It also introduces two new variables: `header-padding` and `header-width-normal` (per Susan).

For some reason or the other, the subnav() mixin is being included. @arelia Please let me know if you wish to exclude this since this is technically not a part of the story.

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-1308

**Changes:**
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before
![header-before](https://user-images.githubusercontent.com/5348098/27612189-ffc2319a-5b5a-11e7-8081-8d57e8fcff95.png)

* After
![header-after](https://user-images.githubusercontent.com/5348098/27612193-03b3d678-5b5b-11e7-8219-7f4cb473ecf8.png)

**QA Links:**
http://web.employer-7.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
  * http://web.employer-7.development.c66.me/recruiting-solutions/display-advertising
  * http://web.employer-7.development.c66.me/recruiting-solutions/talent-driver
  * http://web.employer-7.development.c66.me/recruiting-solutions/source-and-screen
  * http://web.employer-7.development.c66.me/recruiting-solutions/hiring-events
  * http://web.employer-7.development.c66.me/employment-screening/drug-testing-services
  * http://web.employer-7.development.c66.me/employment-screening/electronic-i9-e-verify
  * http://web.employer-7.development.c66.me/recruiting-solutions/job-posting-and-candidate-profile-trends

* Steps to take
  * Make sure the header looks ok. It should have 70px padding top and bottom. It should have a max-width of 650px (per Susan). Beyond that, compare it to the prod versions of the links above. The text should be the same, it should respond the same as well as carry the same properties excluding the refactored padding and width.

* Responsive considerations
  * N/A


**Relevant PRs/Dependencies:**
https://github.com/cbdr/employer/pull/898

**Additional Information**
